resources/js/views/Index.vue

```diff
@@ -2,24 +2,36 @@
     <div>
         <heading class="mb-6">Stripe Dashboard</heading>
 
-        <balance-card></balance-card>
+        <balance-card />
```

Comment:
Just a different syntax for a component! Since we don't have any content _inside_ the component tags, it's generally preferred to do this (it's called `self-closing`).

Here are some more details from the style guide: https://vuejs.org/v2/style-guide/#Self-closing-components-strongly-recommended